distribution/systemd/hardened-vector@.service

```diff
@@ -0,0 +1,55 @@
+[Unit]
+Description=Vector - %i
+Documentation=https://vector.dev
+After=network-online.target local-fs.target remote-fs.target time-sync.target
+Requires=network-online.target local-fs.target remote-fs.target time-sync.target
+
+[Service]
+EnvironmentFile=-/etc/default/vector
+User=vector
+Group=vector
+ExecStartPre=/usr/bin/vector validate --config-toml /etc/vector/%i.toml
+ExecStart=/usr/bin/vector --config-toml /etc/vector/%i.toml
+ExecReload=/usr/bin/vector validate --config-toml /etc/vector/%i.toml
+ExecReload=/bin/kill -HUP $MAINPID
+Restart=no
```

Comment:
This (and the old hardened file) should probably be `always`?

Reply:
I will update the `Restart` value.